pkg/grpc/universalapi/api_workflow.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package universalapi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/dapr/components-contrib/workflows"
+	"github.com/dapr/dapr/pkg/messages"
+	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
+)
+
+func (a *UniversalAPI) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	if in.InstanceId == "" {
+		err := status.Errorf(codes.InvalidArgument, messages.ErrMissingOrEmptyInstance)
```

Comment:
In Universal APIs, we use APIError to return errors so they also include HTTP status codes. Example: 

- Define the errors (changing from string constants to APIError): https://github.com/dapr/dapr/pull/5970/files#diff-c575bdbfe274bf56f437a3127077d0aa30fc43961e7e2482b5ab48a0afb6205bR115-R120
- Use them (with and without format): https://github.com/dapr/dapr/pull/5970/files#diff-98cee75d8c2efb1f1903c19bb4559637e6366d5af4095d7470d0f33e8b89f650R52-R63